builders/manage/.pages

```diff
@@ -3,3 +3,4 @@ hide: false
 nav:
   - index.md
   - sudo
```

Comment:
if we are going to have "Sudo" and "dApp" it should not be named  "Sudo" because we use sudo also for the "dApp"

Maybe we should come up with another name like Developer Portal and Appchain Dashboard?